README.md

```diff
@@ -131,9 +132,34 @@ In case using Vagrant is not desired, ansible playbooks and roles from this repo
 on test.example.com machine, where the dev env should be deployed
 ```sh
 useradd vagrant
+echo vagrant:redhat | chpasswd
 echo "vagrant	ALL=(ALL)	NOPASSWD: ALL" >> /etc/sudoers.d/vagrant
 ```
 
+#### Python Venv for MacOS or Non Ansible Linux
+
+Here we can see the initial configuration, when creating the python virtual environment and preparing the setup (MACOS Users or Regular Linux with no ansible)
+```sh
+/usr/local/bin/python3-intel64 -m venv ~/.venv/forklift
+source ~/.venv/forklift/bin/activate
+pip install --upgrade pip
+pip install ansible
+```
+Note. Above, you can see `python3-intel64`. This is very specific for MacOS M1 or M2 with arm processors, just to keep the compatibility with x86. If you have a regular x86, feel free to use `python3` and it should be enough.
+
+
+Then you can clone forklift and move on
+```sh
+git clone https://github.com/theforeman/forklift.git
+cd forklift
+ansible-galaxy collection install -r requirements.yml
+```
+
+Now, you can share the ssh key with the server
+```sh
+ssh-copy-id vagrant@test.example.com
+```
+
```

Comment:
Is this a repeat of previous steps in our documentation that is unspecific to the virtual env method?

Reply:
ok, let's check how we can do this better. In the requirements, we can see
```
Vagrant - 2.2+ - Both the VirtualBox and Libvirt providers are tested
Ansible - 2.12+
[Vagrant Libvirt provider plugin](https://github.com/vagrant-libvirt/vagrant-libvirt) (if using Libvirt)
Virtualization enabled in BIOS
```

For a regular fedora, we can install the packages and it will work with no issues. As a mac user, trying to test this environment and set the best practices, it's not standard ansible via package, as well vagrant. Once virtualization it's not something that will works in a short term for this kind of processor, what I'm doing is, using the forklift to prepare the basic installation, using the plays that are already around, and installing the minimal as possible.

To achieve that, we can use virtual env, python3 is already standard on mac, that specific binary will keep the compatibility with x86 python modules, then internally we can just install ansible anv voila, we can proceed using the plays.

All the plays that we have atm will work with no problem? No idea, but I'll test! 😉 

With that said, I'm recommending the venv just for the section below, for everyone else with fedora/rhel, it should not be a big deal, they could install the rpms and everything should be fine.
---
#### Python Venv for MacOS or Non Ansible Linux
---

Let me know what you think.
Waldirio